config/sys.config

```diff
@@ -26,7 +26,8 @@
         {random_peer_pred, fun bn_sup:random_val_predicate/1}
     ]},
     {blockchain, [
-        {s3_base_url, "https://snapshots.helium.wtf/mainnet"},
+        {snap_source_base_url, "https://snapshots.helium.wtf/mainnet"},
```

Comment:
We will need to update the specific testnet configs to pull from testnet snapshots (if any)

Reply:
they're marked as:

```
{honor_quick_sync, false}
```